src/server/scripts/EasternKingdoms/BlackrockMountain/MoltenCore/boss_magmadar.cpp

```diff
@@ -11,91 +11,120 @@ SDComment: Conflag on ground nyi
 SDCategory: Molten Core
 EndScriptData */
 
-#include "molten_core.h"
-#include "ObjectMgr.h"
-#include "ScriptedCreature.h"
 #include "ScriptMgr.h"
+#include "ScriptedCreature.h"
+#include "molten_core.h"
+#include "SpellScript.h"
 
 enum Texts
 {
-    EMOTE_FRENZY = 0,
-    EMOTE_SMOLDERING = 0,
-    EMOTE_IGNITE = 1,
+    EMOTE_FRENZY                    = 0,
 };
 
 enum Spells
 {
-    SPELL_FRENZY = 19451,
-    SPELL_MAGMA_SPIT = 19449,
-    SPELL_PANIC = 19408,
-    SPELL_LAVA_BOMB = 19428,
-    SPELL_SERRATED_BITE = 19771,
+    SPELL_FRENZY                    = 19451,
+    SPELL_MAGMA_SPIT                = 19449,
+    SPELL_PANIC                     = 19408,                    // Aoe fear
```

Comment:
```suggestion
    SPELL_PANIC                     = 19408,
```

Reply:
All comments?

Reply:
https://www.azerothcore.org/wiki/cpp-code-standards#comments

In this case, I say leave or remove, doesn't matter